index.html

```diff
@@ -0,0 +1,204 @@
+            English
+          </a>
+        </li>
+        <li class="me-4">
+          <a href="#">
+            My Page
+          </a>
+        </li>
+        <li class="me-70">
+          <a href="#">
+            Log Out
+          </a>
+        </li>
+      </ul>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light pt-0 justify-content-evenly pt-2">
+        <div class="container-fluid">
+          <a class="navbar-brand d-lg-block d-sm-none d-none ms-5" href="/microverse-Capstone1/">
+            <img src="./imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+          </a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse w-100" id="navbarSupportedContent">
+            <ul class="navbar-nav mb-2 mb-lg-0 align-items-center w-100 justify-content-lg-end justify-content-sm-evenly justify-content-evenly me-lg-5">
+              <li class="nav-item d-lg-none d-sm-block d-block">
+                <a class="nav-link navbar-brand w-100" href="/microverse-Capstone1/">
+                  <img src="./imgs/logo.jpg" alt="logo" class="img-fluid"/>
+                </a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="./pages/about.html">About</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">contact</a>
+              </li>
+              <li class="nav-item me-lg-5 p-1">
+                <a class="nav-link text-success" href="#">
+                  Log Out
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Start Headline -->
+    <section class="headline container mt-3">
+      <h1 class="d-block mb-2">
+        "Hello Future Programmers"
+        <span>Creatve Programmers and UI UX Designers</span>
+      </h1>
+      <p class="ms-md-5 ms-3">
+        if you want to be a professional web developer. 
+        So it's time to take our course and become a professional developer. 
+        We are a group of professional teachers.
+      </p>
+    </section>
+    <!-- Start Main Program -->
+    <section class="main-program container-fluid text-light text-center mb-4 py-5">
+      <h2 class="mb-5">
+        Main Program
+      </h2>
+      <div class="row">
```

Comment:
Kindly add the requested design for this section:

**- Requested design:**
![132544753-0c471b40-d361-445f-8133-66606d69e18a](https://user-images.githubusercontent.com/22258274/132662983-ec8236b2-372f-4b45-b8ec-729a47e3e266.png)

#
**- Yours:**
![132544689-05fd1751-6d83-4e41-9ae8-a3a335cca836](https://user-images.githubusercontent.com/22258274/132663077-d48c6179-a421-4b63-83c9-7355855d0a95.png)

Reply:
@carlos-ssh
Sir, Do you mean the content?

Reply:
Resolved this!